x-pack/platform/packages/shared/kbn-streams-schema/src/models/group/index.ts

```diff
@@ -11,14 +11,43 @@ import { validation } from '../validation/validation';
 import type { ModelValidation } from '../validation/model_validation';
 import { modelValidation } from '../validation/model_validation';
 
+export type GroupStreamRelationshipType = 'related' | 'child' | 'parent' | 'dependency';
+
+interface StreamRelationship {
+  name: string;
+  type: GroupStreamRelationshipType;
+}
+
+const streamRelationshipSchema = z.object({
+  name: z.string(),
+  type: z.union([
+    z.literal('related'),
+    z.literal('child'),
+    z.literal('parent'),
+    z.literal('dependency'),
+  ]),
+});
+
 export interface Group {
-  members: string[];
+  owner: string;
+  tier: 1 | 2 | 3 | 4;
+  tags: string[];
+  runbook_links: string[];
+  documentation_links: string[];
+  repository_links: string[];
+  relationships: StreamRelationship[];
```

Comment:
Why do we need all this stuff? This seems way too specific?

Reply:
Had another round with PM and we decided to cut this in favor of the metadata field that's being added [here](https://github.com/elastic/kibana/pull/232802).
We'll revert the relationships back to just `members` and treat Group streams as mainly "collections" for the first milestone.
The tags we'll keep because we want tags for all Streams later.